### ISSUE TITLE
LoadError: InitError: could not load library "/cache/julia-buildkite-plugin/depots/5b300254-1738-4989-ae0a-f4d2d937f953/artifacts/cb7fc2801ca0133a5bdea4bc4585d07c08284cfa/lib/libsundials_sunlinsolklu.so"

### ISSUE THREAD

**Issue by ParamThakkar123** [SciML/Sundials.jl#465]
**Describe the bug 🐞**

Sundials.jl gives the following load error on download

![image](https://github.com/user-attachments/assets/4c088217-d705-48e4-91b3-f9bca57a4d8f)

```julia
<html>
<body>
<!--StartFragment-->
ERROR: LoadError: InitError: could not load library "/cache/julia-buildkite-plugin/depots/5b300254-1738-4989-ae0a-f4d2d937f953/artifacts/cb7fc2801ca0133a5bdea4bc4585d07c08284cfa/lib/libsundials_sunlinsolklu.so"
--
  | libklu.so.1: cannot open shared object file: No such file or directory
  | Stacktrace:
  | [1] dlopen(s::String, flags::UInt32; throw_error::Bool)
  | @ Base.Libc.Libdl ./libdl.jl:117
  | [2] dlopen(s::String, flags::UInt32)
  | @ Base.Libc.Libdl ./libdl.jl:116
  | [3] macro expansion
  | @ /cache/julia-buildkite-plugin/depots/5b300254-1738-4989-ae0a-f4d2d937f953/packages/JLLWrappers/GfYNv/src/products/library_generators.jl:63 [inlined]
  | [4] __init__()
  | @ Sundials_jll /cache/julia-buildkite-plugin/depots/5b300254-1738-4989-ae0a-f4d2d937f953/packages/Sundials_jll/rXn1i/src/wrappers/x86_64-linux-gnu-libgfortran5.jl:93
  | [5] run_module_init(mod::Module, i::Int64)
  | @ Base ./loading.jl:1188
  | [6] register_restored_modules(sv::Core.SimpleVector, pkg::Base.PkgId, path::String)
  | @ Base ./loading.jl:1176
  | [7] _include_from_serialized(pkg::Base.PkgId, path::String, ocachepath::String, depmods::Vector{Any})
  | @ Base ./loading.jl:1121
  | [8] _require_search_from_serialized(pkg::Base.PkgId, sourcepath::String, build_id::UInt128)
  | @ Base ./loading.jl:1644
  | [9] _require(pkg::Base.PkgId, env::String)
  | @ Base ./loading.jl:2001
  | [10] __require_prelocked(uuidkey::Base.PkgId, env::String)
  | @ Base ./loading.jl:1875
  | [11] #invoke_in_world#3
  | @ ./essentials.jl:926 [inlined]
  | [12] invoke_in_world
  | @ ./essentials.jl:923 [inlined]
  | [13] _require_prelocked(uuidkey::Base.PkgId, env::String)
  | @ Base ./loading.jl:1866
  | [14] macro expansion
  | @ ./loading.jl:1853 [inlined]
  | [15] macro expansion
  | @ ./lock.jl:267 [inlined]
  | [16] __require(into::Module, mod::Symbol)
  | @ Base ./loading.jl:1816
  | [17] #invoke_in_world#3
  | @ ./essentials.jl:926 [inlined]
  | [18] invoke_in_world
  | @ ./essentials.jl:923 [inlined]
  | [19] require(into::Module, mod::Symbol)
  | @ Base ./loading.jl:1809
  | [20] include
  | @ ./Base.jl:495 [inlined]
  | [21] include_package_for_output(pkg::Base.PkgId, input::String, depot_path::Vector{String}, dl_load_path::Vector{String}, load_path::Vector{String}, concrete_deps::Vector{Pair{Base.PkgId, UInt128}}, source::String)
  | @ Base ./loading.jl:2285
  | [22] top-level scope
  | @ stdin:3
  | during initialization of module Sundials_jll
```

</body>
</html>

This error occurs in the CI in SciMLBenchmarks.jl repository. While it works fine locally on Julia version 1.9.1

**Expected behavior**

**Minimal Reproducible Example 👇**

Without MRE, we would only be able to help you to a limited extent, and attention to the issue would be limited. to know more about MRE refer to [wikipedia](https://en.wikipedia.org/wiki/Minimal_reproducible_example) and [stackoverflow](https://stackoverflow.com/help/minimal-reproducible-example).

```julia

```

**Error & Stacktrace ⚠️**

```julia

```

**Environment (please complete the following information):**

  - Output of `using Pkg; Pkg.status()`

```julia

```

  - Output of `using Pkg; Pkg.status(; mode = PKGMODE_MANIFEST)`

```julia

```

  - Output of `versioninfo()`

```julia

```

**Additional context**

**Comment by ChrisRackauckas** [comment]
Reproducer?

**Comment by ParamThakkar123** [comment]
> Reproducer?

Actually this error occurred during CI tests of SciMLBenchmarks.jl in ComplicatedPDE benchmarks. Julia version 1.9.1 and Sundials version 4 and above. PR #1146

**Comment by ChrisRackauckas** [comment]
SciMLBenchmarks does not run on v1.9.

**Comment by ParamThakkar123** [comment]
> SciMLBenchmarks does not run on v1.9.

Yeah. ComplicatedPDE runs on 1.9. As you said, in PR #1146 on SciMLBenchmarks I will update complicatedPDE to 1.10 and see if I get the error again

**Comment by ParamThakkar123** [comment]
The issue has been resolved on updating the Julia version to 1.10. closing this one
